jsf/plugins/org.eclipse.jst.pagedesigner/src/org/eclipse/jst/pagedesigner/converter/AbstractTagConverter.java

```diff
@@ -362,7 +362,7 @@ protected String mapValue(String value) {
 		} else {
 			// preview mode. let's try to display the value.
 			try {
-				//Bug 319317 - Third-party plug-in providing javax.servlet.jsp.el version 2.1 or greater breaks WPE preview
+				//Bug 319317 - Third-party plug-in providing jakarta.el version 2.1 or greater breaks WPE preview
```

Comment:
This refers to the title of the issue, https://bugs.eclipse.org/bugs/show_bug.cgi?id=319317 .

AFAIK at least a copy of the bugzilla issues will be kept. Therefore, this comment will probably still useful to find the original issue and should not be changed.

Reply:
change is [reverted](https://github.com/eclipse-jsf/webtools.jsf/pull/9/commits/61c1cb00662c35bde51b31b272c8b43b539ac778).